### DISCUSSION TITLE
UGS Desktop Shortcut

### DISCUSSION THREAD

**Discussion by brysoncotton** [winder/Universal-G-Code-Sender#1682] [Help]
Does anyone know how to create a desktop shortcut for UGS on Raspberry Pi OS? Thanks

**Comment by brysoncotton** [comment]
NVM I figured it out =/

**Reply by mcinquino** [reply]
I was able to create a script to run it. I have had no luck giving it a custom icon however. None of the methods I could find worked. Have you given your shortcut a icon? 

Also I would like to have UGS run on startup. I can't get that work on RPi either.
